locales/base/translation.json

```diff
@@ -1694,8 +1694,8 @@
     "swapFrom": "Swap from",
     "swapTo": "Swap to",
     "rate": "Rate",
-    "estimatedFee": "Estimated Fee",
-    "viewOnExplorer": "View on CeloExplorer"
```

Comment:
Are older client stilling using this string? If so we should keep it for backward compatibility.

Reply:
Good catch, thanks! Restored it.